json-schemas/iva/defects/get/index.json

```diff
@@ -140,6 +121,17 @@
 					}
 				}
 			}
+		},
+		"additionalInformation": {

```

Comment:
Not sure this should be on the GET schema? Isn't this only going to be used when FE are saving IVA defects on a test result?

Reply:
The additional info on regular defects comes back in the response, wouldn't we want it to function similarly?

Reply:
I am in favour of returning everything, by not returning this we're making a decision for clients that they don't need it.